docs/examples/visualize_quality_metrics.ipynb

```diff
@@ -114,9 +113,17 @@
-06', 'timestamp': '2024-11-19 04:55:27.396209+00:00'}, 'statistics': {'average': 1.762230540603657e-05, 'median': 1.6176355888197993e-05}}, 'prx_rb_fidelity': {'QB1': {'value': '0.9982569828378964', 'unit': '', 'uncertainty': '4.4558738160126356e-05', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'drag_gaussian', 'parameter': 'par=d2'}, 'QB2': {'value': '0.9983649180842389', 'unit': '', 'uncertainty': '5.6303748003846376e-05', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'drag_gaussian', 'parameter': 'par=d2'}, 'QB3': {'value': '0.9973653607212306', 'unit': '', 'uncertainty': '0.00010159568802245763', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'drag_gaussian', 'parameter': 'par=d2'}, 'QB4': {'value': '0.9949507106177625', 'unit': '', 'uncertainty': '0.0001750026245526004', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'drag_gaussian', 'parameter': 'par=d2'}, 'QB5': {'value': '0.9973544325850584', 'unit': '', 'uncertainty': '9.469127928794496e-05', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'drag_gaussian', 'parameter': 'par=d2'}, 'statistics': {'average': 0.9972584809692373, 'median': 0.9973653607212306}}, 'clifford_rb_fidelity': {'QB1__QB3': {'value': '0.9562542551687688', 'unit': '', 'uncertainty': '0.0008895885711898765', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'xy_cz', 'parameter': 'par=d2'}, 'QB3__QB5': {'value': '0.953821355189975', 'unit': '', 'uncertainty': '0.0008772377287973878', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'xy_cz', 'parameter': 'par=d2'}, 'QB2__QB3': {'value': '0.8950990408487102', 'unit': '', 'uncertainty': '0.0020508985394268437', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'xy_cz', 'parameter': 'par=d2'}, 'QB3__QB4': {'value': '0.9439582461347993', 'unit': '', 'uncertainty': '0.00119987380040661', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'xy_cz', 'parameter': 'par=d2'}, 'statistics': {'average': 0.9372832243355633, 'median': 0.9488898006623872}}, 'cz_irb_fidelity': {'QB1__QB3': {'value': '0.9855008776293077', 'unit': '', 'uncertainty': '0.0015238326332719856', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'tgss', 'parameter': 'par=d2'}, 'QB3__QB5': {'value': '0.9854688635058743', 'unit': '', 'uncertainty': '0.0014956904073850882', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'tgss', 'parameter': 'par=d2'}, 'QB2__QB3': {'value': '0.9439826241533328', 'unit': '', 'uncertainty': '0.003954673659719742', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'tgss', 'parameter': 'par=d2'}, 'QB3__QB4': {'value': '0.9805787195442951', 'unit': '', 'uncertainty': '0.002085397728623927', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'tgss', 'parameter': 'par=d2'}, 'statistics': {'average': 0.9738827712082024, 'median': 0.9830237915250847}}}}\n"
```

Comment:
I would prefer that we clear the output before merging

Reply:
I agree